nucypher/characters/lawful.py

```diff
@@ -1316,9 +1323,8 @@ def run(self,
             stdio.StandardIO(UrsulaCommandProtocol(ursula=self, emitter=emitter))
 
         if hendrix:
-
             if emitter:
-                emitter.message(f"Starting Ursula on {self.rest_interface}", color='green', bold=True)
+                emitter.message(f"✓ Rest Server https://{self.rest_interface}", color='green')
```

Comment:
```suggestion
                emitter.message(f"✓ Starting REST Server on https://{self.rest_interface}", color='green')
```

Reply:
Lately I've been feeling more and more that REST/ReST is a misnomer in our codebase, particular from a UX perspective.  I suggest we start moving towards *HTTP Server* instead.

Reply:
Yes.  We aren't exactly dogmatically adhering to REST.